.ci/validation/src/index.ts

```diff
@@ -24,7 +24,7 @@ export module SWSchemaValidator {
   const ajv = new Ajv({ strict: false, allowUnionTypes: true });
   addFormats(ajv);
 
-  const workflowSchemaId = "https://serverlessworkflow.io/schemas/1.0.0/workflow.yaml";
+  const workflowSchemaId = "https://serverlessworkflow.io/schemas/1.0.1/workflow.yaml";
```

Comment:
Are we merging this as part of the 1.0.1 release?

Reply:
I'm not sure what else is planned for v1.0.1, but since this introduces a schema change, I think we should bump the version. Otherwise, anyone relying on v1.0.0 might be caught off guard when this gets merged.
For example, if @fjtirado wasn’t aware of the change and we published an override of v1.0.0, it could break the Java SDK.